includes/motherduck_destination_server.hpp

```diff
@@ -15,6 +15,10 @@ static constexpr const char *const CONFIG_TEST_NAME_AUTHENTICATE =
 static constexpr const char *const CONFIG_TEST_NAME_CSV_BLOCK_SIZE =
     "test_csv_block_size";
 
+static const int DUCKDB_DEFAULT_PRECISION = 18;
+
+static const int DUCKDB_DEFAULT_SCALE = 3;
```

Comment:
Nit: this should probably be in the cpp file.  Otherwise we'll actually have multiple copies of these constants in every cpp file that includes this one.

Reply:
won't `#pragma once` prevent that?